packages/explorer/src/components/ConnectButton.tsx

```diff
@@ -0,0 +1,68 @@
+import { PlugIcon, ZapIcon } from "lucide-react";
+import { anvil } from "viem/chains";
+import { ConnectButton as RainbowConnectButton } from "@rainbow-me/rainbowkit";
+import { cn } from "../lib/utils";
+import { AccountSelect } from "./AccountSelect";
+import { Button, ButtonProps } from "./ui/Button";
+
+export function StyledConnectButton({ children, ...props }: ButtonProps) {
+  return (
+    <Button type="button" size="sm" {...props}>
```

Comment:
is this doing enough to justify its own component?

Reply:
not sure tbh, the main purpose is to keep styling consistent but since it's all in one file, perhaps unneeded indeed